sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -443,12 +445,56 @@ void CAFMaker::beginRun(art::Run& run) {
 void CAFMaker::beginSubRun(art::SubRun& sr) {
   // get the POT
   // get POT information
-  art::Handle<sumdata::POTSummary> pot_handle;
-  sr.getByLabel("generator", pot_handle);
-
-  if (pot_handle.isValid()) {
-    fSubRunPOT = pot_handle->totgoodpot;
-    fTotalPOT += fSubRunPOT;
+  fIsRealData = true;
+  if(fIsRealData)
+  {
```

Comment:
Can we pull this out into a separate `FillBNBPOTInfo` function? Maybe even in a new `FillExposure.cxx`? Hopefully we will soon also have a NuMI version of this, which is going to make beginSubRun very long.

Reply:
Sure, that was actually my first thought. I will work towards that